src/lib/src/component/tree-viewer/indicator.box.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Workspace } from '../../common/workspace';
+import { MarkerState } from '../../common/markers/marker.state';
+
+@Component({
+  selector: 'indicator-box',
+  templateUrl: './indicator.box.component.html',
+  styleUrls: ['./indicator.box.component.css']
+})
+export class IndicatorBoxComponent {
+  @Input() model: { workspace: Workspace, path: string, states: MarkerState[] };
+
+  get cssClasses(): string {
```

Comment:
TO DISCUSS: we should generally put some docstring on public methods, shouldn't we?